src/mower_comms/src/mower_comms.cpp

```diff
@@ -107,7 +107,7 @@ void publishActuators() {
     }
 
     if(mow_xesc_interface) {
-        mow_xesc_interface->setDutyCycle(speed_mow);
+        mow_xesc_interface->setDutyCycle(speed_mow*((rand() % 2) * 2 - 1));
```

Comment:
I was about to create a PR for this, but set an initial direction at random and then just flipped it each time  setMowEnabled is called

Reply:
I think this would be flipping direction every time publishActuators is called, 20ms IIRC

Reply:
I pushed this to my mower yesterday, along with a tacho monitoring. I activated the mower by calling setMowEnabled and afaik it just worked. But yeah, if publishActuators is called every 20ms, than this is not the way to go.